sound/soc/sof/intel/shim.h

```diff
@@ -186,6 +186,7 @@ struct sof_intel_dsp_desc {
 	enum sof_intel_hw_ip_version hw_ip_version;
```

Comment:
Probably change the commit message to `DSP core power down sequences are different between CAVS platforms and ACE platforms.`